src/js/components/Notification/Notification.js

```diff
@@ -45,39 +90,92 @@ const Notification = ({ message, onClose, id, status, title, toast }) => {
   ]);
 
   const { icon: CloseIcon } = theme.notification.close;
-  const { icon: StatusIcon, color } = theme.notification[status];
+  const { icon: StatusIcon, background, color } = theme.notification[status];
   const { color: closeIconColor } = theme.notification.close;
 
+  const { direction } = toast ? theme.notification.toast : theme.notification;
+
+  const TextWrapper = direction === 'row' ? Text : Fragment;
+
+  // notification is built with two child boxes that contain:
+  // 1. icon + text (wrapped in button when clickable)
+  // 2. close button
+  // pad needs to be applied to the child boxes, but we don't want to apply
+  // extra padding between the icon + text and the button.
+  const { pad } = theme.notification.container;
+  let textPad;
+  let closeButtonPad;
+  if (onClose) [textPad, closeButtonPad] = adaptThemeStyle(pad, theme);
+  else textPad = pad;
+
+  let actions;
+  let message = messageProp;
+
+  if (actionsProp)
+    actions = actionsProp.map((action, index) => (
+      <NotificationAnchor
+        key={action.label}
+        // create space between first anchor and text content
+        margin={
```

Comment:
Wondering if we can just put a "&nbsp;" after `{message}` below and avoid having to muck with the `margin` here.

Reply:
To my eye, the space came across a bit too small -- I can screenshot a version of each side by side. Might just be my opinion

Reply:
With some margin
<img width="1427" alt="Screen Shot 2022-03-02 at 9 39 45 AM" src="https://user-images.githubusercontent.com/12522275/156417607-012964c8-c168-45eb-b3c9-b8b69c94233b.png">

With just a space (feel like it looks squished)
<img width="1428" alt="Screen Shot 2022-03-02 at 9 40 12 AM" src="https://user-images.githubusercontent.com/12522275/156417621-8467487f-71ed-4a84-a818-47ebef2dd07f.png">

Reply:
agreed with your margin